### PR TITLE
bump gffutils to v0.10.1

### PR DESCRIPTION
**Pull request by daler**
Bioconda requires reviews prior to merging pull-requests (PRs). To facilitate this, once your PR is passing tests and ready to be merged, please add the `please review & merge` label so other members of the bioconda community can have a look at your PR and either make suggestions or merge it.

* [X] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/contributor/guidelines.html).
* [ ] This PR adds a new recipe.
* [ ] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [X] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

 * `@BiocondaBot please update` will cause the BiocondaBot to merge the master branch into a PR
 * `@BiocondaBot please add label` will add the `please review & merge` label.
 * `@BiocondaBot please fetch artifacts` will post links to packages and docker containers built by the CI system. You can use this to test packages locally before merging.

For members of the Bioconda project, the following command is also available:

 * `@BiocondaBot please merge` will cause packages/containers to be uploaded and a PR merged. Someone must approve a PR first! This has the benefit of not wasting CI build time required by manually merging PRs.

If you have questions, please post them in gitter or ping `@bioconda/core` in a comment (if you are not able to directly ping `@bioconda/core` then the bot will repost your comment and enable pinging).

### CODE REVIEW

**Comment by daler** [comment]
@BiocondaBot please add label

**Comment by npavlovikj** [comment]
@BiocondaBot please fetch artifacts

File: recipes/gffutils/meta.yaml [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
 
 build:
   noarch: python
-  number: 1
+  number: 0
 
 source:
-  url: https://pypi.python.org/packages/b4/ea/54ca403a8d471849606fb432b9afcf73c8c5105ea2dd87b8d38bd5217c5a/gffutils-{{ version }}.tar.gz
-  sha256: e1b0bf2b422ec3491be13d77bf7baf71d274abd844e8f8b6c1217a95b236218b
+  url: https://files.pythonhosted.org/packages/87/d6/f893f14fd6cf1c9e758259f6e0f9ba1ae03370988d8f1a3ea8d790ca5b06/gffutils-0.10.1.tar.gz
```
Comment (npavlovikj) [inline recipes/gffutils/meta.yaml]: The url should probably be `https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz`.

**Comment by npavlovikj** [comment]
@BiocondaBot please fetch artifacts

**Comment by npavlovikj** [comment]
@BiocondaBot please merge

**Pull request merged**